src/planwise/client/projects/db.cljs

```diff
@@ -35,7 +35,13 @@
    :map-view       {} ;; {:keys position zoom}
    :demand-map-key nil ;; string
    :unsatisfied-count nil ;; number
-   :project-data   {}}) ;; {:keys id goal region-id stats filters region-population region-area-km2}
+   :project-data   {} ;; {:keys id goal region-id stats filters region-population region-area-km2}
+   :wizard         {
+                    :set false
+                    :tabs {
+                      :demographics    :unvisited
+                      :facilities      :unvisited
+                      :transport       :unvisited}}})
```

Comment:
I'm not sure I like the model for the wizard. Typically wizards are modelled using the _step_ the user is currently in. So `:wizard` could be simply one of `:demographics`, `:facilities`, `:transport`, `:finished`; and then a _subscription_ could answer whether a particular tab should be displayed as visited or not, based on the current step.